etc/wpt-summarize.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
+
+# Copyright 2019 The Servo Project Developers. See the COPYRIGHT
+# file at the top-level directory of this distribution.
+#
+# Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+# http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+# <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+# option. This file may not be copied, modified, or distributed
+# except according to those terms.
+
+import sys
+import json
+
+full_search = len(sys.argv) > 3 and sys.argv[3] == '--full'
+
+with open(sys.argv[1]) as f:
+    data = f.readlines()
+    thread = None
+    for entry in data:
+        entry = json.loads(entry)
+        if thread and "thread" in entry:
+            if entry["thread"] == thread:
+                print(json.dumps(entry))
```

Comment:
won't this skip test_starts from other threads?

Reply:
I'm not sure I understand the concern here. The purpose of this script is to extract the output of a single test, so we don't care about other test_start events.

Reply:
oh! i think i hadn't fully understood that part